include/godot_cpp/variant/color_names.inc.hpp

```diff
@@ -186,3 +191,5 @@ static NamedColor named_colors[] = {
 };
 
 } // namespace godot
+
+#endif
```

Comment:
Missing newline at EOF but I think the guards might not be needed, this is not a header.

Reply:
Done in 0b3b1c4491b332738ac72c7b7d1ccd634b3a852b